[W14-1] ClinicIO

arsalan0c opened the pull request:
@aaronseahyh 
@gingivitiss 
@iamjackslayer 
@jjlee050

nhs-work reviewed (commented):
Good job on updating the documentation! Please keep this PR open!

However, do take note of the following points:

-Do update all instances of addressbook in the User Guide and the Developer Guide to ClinicIO

-Please add in the new commands specific to ClinicIO for the User Guide

-Update the project scope and user stories for the User Guide

-How is viewing and finding a patient’s record different? Would I need to find all of a patient’s records before I can view one of them?

-Consider adding find a reference to the patient record use case in the edit/delete patient record use case

-Do note the incorrect formatting for the extension for D.5, the flow of events after the extension should each be on a new line

Please do start working towards v1.1. Have a good recess week!

damithc commented:
Good job on following the guidelines for team member photos closely. Your team is one of the very few to have done it correctly so far. 👍

damithc commented:
Also, good job on figuring out how to tweak the top nav bar to fit your product. 👍 
I encourage you (no obligation) to share with others how to do it through a post in our forum. I suspect some others will figure it out eventually too, but if you share it first, you earn the bragging rights and it helps your own brand. i.e., Only strong teams are able to share their knowledge with others without fear of competition.

nhs-work commented:
Good job on finishing v1.1! Please continue keeping the PR open.

Please take note of the following points:

-Please include all the features that will be used for ClinicIO, the queue features are currently missing.

-If a feature has yet to be implemented in the current version, you should mark it as `coming in v2.0`.

-Formatting errors under Command Summary:

![image](https://user-images.githubusercontent.com/35650824/46574180-47387e80-c9d2-11e8-85d8-5faf46700c84.png)

-Why is able to handle multiple doctors under target user profile?

![image](https://user-images.githubusercontent.com/35650824/46574092-d9d81e00-c9d0-11e8-925c-6d0e6b7342ec.png)

-Well done for the use cases, you have covered a good range of features and have been quite thorough with the extensions.

-However, do consider for D.4. Use case: Delete patient record, what if the user deletes the patient record of a patient that is currently in the queue?

-For D.5. Use case: Assign patient to a different queue for each room, extension 2a has not been elaborated upon.

-Please remove Appendix G: Product Survey if you are not intending to use it.

nhs-work commented:
Good job on finishing v1.2!👍 Please continue keeping the PR open.

Please take note of the following points:

**User Guide**
-To keep the it consistent, commands that appear in the command summary should also have their own section under features especially for key features. Please do include a short write up and example for some of the missing features like queue.

**Developer Guide**
- Note that some images are not being rendered correctly:

![image](https://user-images.githubusercontent.com/35650824/47257307-a01e1180-d4be-11e8-877c-0ed5099bad8f.png)

- Feel free to remove this:

![image](https://user-images.githubusercontent.com/35650824/47257327-d2c80a00-d4be-11e8-9dda-f026bc86204c.png)

Update existing diagrams where applicable:

![image](https://user-images.githubusercontent.com/35650824/47257344-f2f7c900-d4be-11e8-8cfe-144d4b46eda2.png)
 

- Each member is required to describe the implementation of at least one enhancement they are adding, however there are only two implementation descriptions currently (appointment and medicine inventory). Please add in your description if you have not done so as soon as possible.

- Good job on adding the design descriptions for appointment and medicine inventory. But there are not enough new UML diagrams to give feedback on your UML modelling skills. Try to add more soon.

Please update the above mentioned points and work towards getting the product to a user-testable state by v1.3 as soon as possible as your team seems a little behind right now 😟.

jackedelic committed:
polish ppp

nhs-work commented:
Good job on finishing mid v1.4!👍 Please continue keeping the PR open. Just a few more weeks to go, keep up the good work!

Please take note of the following points for your diagrams:

**Queue:**
Please add in one of the many UML diagrams we have taught in the course so that we can assess your ability to utilise these diagrams.

**Medicine Inventory:**
Please add in one of the many UML diagrams we have taught in the course so that we can assess your ability to utilise these diagrams.

**Analytics:**
Do note that for class diagrams, the attributes are of the form name:type.
Do also note the text is too small.

**Export:**

![image](https://user-images.githubusercontent.com/35650824/48137934-1e135280-e2de-11e8-9f78-825e82244661.png)

Note that the activity diagram captures the actions and control flows of the activity. You should use actions such as “User chooses to export patients”

![image](https://user-images.githubusercontent.com/35650824/48137947-24a1ca00-e2de-11e8-829f-2129e5273967.png)

Instead of having 3 start and end nodes in one diagram, consider either splitting into 3 diagrams or having 1 start node, which then goes into 3 alternate paths.
Note that isValidLine should not be allowed to split into alternate paths (what kind of a node is used to show the start of an alternate path? What other information is required to determine which path it goes to?)
exportPatients / exportAppointments / exportConsultations / Execute should not be allowed to split into multiple paths as well.

![image](https://user-images.githubusercontent.com/35650824/48137961-31beb900-e2de-11e8-94ad-468c62d24c10.png)

Small mistakes like forgetting the arrowhead as well.

**Login:**
For your sequence diagram, note that we discourage low level diagrams that span multiple components. Try to draw diagrams at multiple levels: one that shows component level interactions and others that show low level interactions within one component.

![image](https://user-images.githubusercontent.com/35650824/48137991-3e431180-e2de-11e8-8dc7-8a368f7e9d02.png)

It appears as though nothing happens even if a record is not found, are there no error messages shown/early termination? Similar issue with the verification of password and the case where user is not a doctor nor a receptionist.

jackedelic committed:
fix images displayed

jackedelic committed:
fix format

jackedelic committed:
add newline at EOF

jackedelic committed:
use include::path feature

jackedelic committed:
add newline at EOF

jackedelic committed:
add reposense link

aaronseahyh committed:
Merge pull request #252 from aaronseahyh/loginRequiredForMedCommands

Require login for Medicine Commands

gingivitiss committed:
fix uniqueappointmentlist

jackedelic committed:
Merge branch 'master' into queue

gingivitiss committed:
Add typical appts.

jackedelic committed:
merge-resolve with aaron's pr

jackedelic committed:
merge-resolve aaron's pr

gingivitiss committed:
Fix checkstyle

jackedelic committed:
fix style

jackedelic committed:
reduce some contents

jackedelic committed:
delete irrelevant file

jackedelic committed:
fix style

jackedelic committed:
delete irrelevant file

jjlee050 committed:
Add ListPatientCommand to ClinicIOParser.

Signed-off-by: jjlee050 <josephlee050@gmail.com>

jjlee050 committed:
Fix checkstyle errors.

Signed-off-by: jjlee050 <josephlee050@gmail.com>

jjlee050 committed:
Add test cases for listpatientcommand.

Signed-off-by: jjlee050 <josephlee050@gmail.com>

jjlee050 committed:
Add PatientNameContainsKeywordsPredicate and Test

Signed-off-by: jjlee050 <josephlee050@gmail.com>

jjlee050 committed:
Fix checkstyle error.

Signed-off-by: jjlee050 <josephlee050@gmail.com>

jjlee050 committed:
Add FindPatientCommand to ClinicIO

Signed-off-by: jjlee050 <josephlee050@gmail.com>

jjlee050 committed:
Check is login for FindPatientCommand.

Signed-off-by: jjlee050 <josephlee050@gmail.com>

jjlee050 committed:
FindPatientCommand.java: Fix compile errors.

Signed-off-by: jjlee050 <josephlee050@gmail.com>

jjlee050 committed:
Add test cases and fix checkstyle error.

Signed-off-by: jjlee050 <josephlee050@gmail.com>

jjlee050 committed:
Fix checkstyle error

Signed-off-by: jjlee050 <josephlee050@gmail.com>

jjlee050 committed:
Add FindPatientCommandTest.java

Signed-off-by: jjlee050 <josephlee050@gmail.com>

jjlee050 committed:
FindPatientCommandTest.java: Fix checkstyle error.

Signed-off-by: jjlee050 <josephlee050@gmail.com>

jjlee050 committed:
Edit version  v1.3 to v1.4

Signed-off-by: jjlee050 <josephlee050@gmail.com>

jjlee050 committed:
Merge pull request #261 from jjlee050/add-listpatient

Implement List and Find Patient

gingivitiss committed:
Edit UG and DG

jjlee050 committed:
Merge pull request #262 from jjlee050/edit-version

Edit version  v1.3 to v1.4

jjlee050 committed:
Edit user guide to fix dry bug and add implemented command.

Signed-off-by: jjlee050 <josephlee050@gmail.com>

jjlee050 committed:
UserGuide.adoc: Edit FAQ.

Signed-off-by: jjlee050 <josephlee050@gmail.com>

jjlee050 committed:
jjlee050.adoc: Edit overview.

Signed-off-by: jjlee050 <josephlee050@gmail.com>

jjlee050 committed:
jjlee050.adoc: Edit major enhancement.

Signed-off-by: jjlee050 <josephlee050@gmail.com>

jjlee050 committed:
jjlee050.adoc: Edit overview.

Signed-off-by: jjlee050 <josephlee050@gmail.com>

jjlee050 committed:
jjlee050.adoc: Edit major enhancements.

Signed-off-by: jjlee050 <josephlee050@gmail.com>

jjlee050 committed:
jjlee050.adoc: Edit minor enhancements.

Signed-off-by: jjlee050 <josephlee050@gmail.com>

jjlee050 committed:
Edit contributions.

Signed-off-by: jjlee050 <josephlee050@gmail.com>

jjlee050 committed:
Fix checkstyle error.

Signed-off-by: jjlee050 <josephlee050@gmail.com>

jjlee050 committed:
Merge pull request #263 from jjlee050/edit-ug

Edit user guide

jjlee050 committed:
Merge pull request #241 from jjlee050/edit-ppp

Update PPP

jjlee050 committed:
Merge pull request #240 from jjlee050/edit-dg

Update DeveloperGuide

aaronseahyh committed:
Added ui components for Medicine.

aaronseahyh committed:
Added ui resources for Medicine.

aaronseahyh committed:
Added ui event for Medicine.

aaronseahyh committed:
Added image for Medicine tabpane.

aaronseahyh committed:
Added handles for Medicine tests.

aaronseahyh committed:
[GuiTestAssert.java] Added implementation for Medicine.

aaronseahyh committed:
Edited MainWindow to include Medicine ui.

aaronseahyh committed:
Added tests for Medicine ui.

jackedelic committed:
edit ppp

jackedelic committed:
delete irrelevant file

jjlee050 committed:
Add images for login sequence diagram

Signed-off-by: jjlee050 <josephlee050@gmail.com>

arsalan0c committed:
Refactor analytics ui (#254)

* add v2.0 to ug

* fix import error

* refactor analytics

* remove unused import

aaronseahyh committed:
Fixed small errors.

jjlee050 committed:
Edit activity diagram for login.

Signed-off-by: jjlee050 <josephlee050@gmail.com>

jjlee050 committed:
Edit login implementation.

Signed-off-by: jjlee050 <josephlee050@gmail.com>

jjlee050 committed:
Add line breaks.

Signed-off-by: jjlee050 <josephlee050@gmail.com>

jjlee050 committed:
Upload broken images.

Signed-off-by: jjlee050 <josephlee050@gmail.com>

arsalan0c committed:
fix switch statement

jjlee050 committed:
Edit activity diagram.

Signed-off-by: jjlee050 <josephlee050@gmail.com>

aaronseahyh committed:
Fixed Checkstyle violations.

jjlee050 committed:
Fix spaces issues.

Signed-off-by: jjlee050 <josephlee050@gmail.com>

jjlee050 committed:
DeveloperGuide.adoc: Add spaces.

Signed-off-by: jjlee050 <josephlee050@gmail.com>

jjlee050 committed:
DeveloperGuide.adoc: Add spaces.

Signed-off-by: jjlee050 <josephlee050@gmail.com>

jjlee050 committed:
DeveloperGuide.adoc: Edit patient management.

Signed-off-by: jjlee050 <josephlee050@gmail.com>

jjlee050 committed:
DeveloperGuide.adoc: Remove spaces.

Signed-off-by: jjlee050 <josephlee050@gmail.com>

aaronseahyh committed:
Fixed Checkstyle violations.

jjlee050 committed:
Merge branch 'master' of https://github.com/CS2103-AY1819S1-W14-1/main into edit-ui

Signed-off-by: jjlee050 <josephlee050@gmail.com>

# Conflicts:
#	src/main/java/seedu/clinicio/ui/MainWindow.java

jjlee050 committed:
Merge pull request #265 from aaronseahyh/medicineUI

Medicine UI

arsalan0c committed:
Merge pull request #266 from arsalanc-v2/analytics4

Fix switch statement

jjlee050 committed:
Merge branch 'master' of https://github.com/CS2103-AY1819S1-W14-1/main into edit-ui

Signed-off-by: jjlee050 <josephlee050@gmail.com>

# Conflicts:
#	src/main/java/seedu/clinicio/ui/MainWindow.java

gingivitiss committed:
Edit DG

arsalan0c committed:
Update documentation (#260)

* update ppp

* update docs

* remove unnecessary diagram

* update export activity diagram

* update export activity diagram

* update ugdg

* update ugdg

* update ugdg

* update ugdg

* update ug

* update ug

* update ppp

* update ppp

* update ugdg

* update ugdg

* update docs

* update ppp

* update ugdg

* update ugdg

* update ppp

* update ppp

* update docs

jackedelic committed:
merge with jjlee's pr251

jjlee050 committed:
Fix UI bug and add scroll pane for analytics.

Signed-off-by: jjlee050 <josephlee050@gmail.com>

jjlee050 committed:
Add ScrollPane to MainWindow instead.

Signed-off-by: jjlee050 <josephlee050@gmail.com>

jjlee050 committed:
Fix scroll pane at MainWindow.

Signed-off-by: jjlee050 <josephlee050@gmail.com>

jjlee050 committed:
Add back alignment

Signed-off-by: jjlee050 <josephlee050@gmail.com>

jjlee050 committed:
Fix checkstyle errors.

Signed-off-by: jjlee050 <josephlee050@gmail.com>

jjlee050 committed:
Fix checkstyle errors.

Signed-off-by: jjlee050 <josephlee050@gmail.com>

jjlee050 committed:
Fix checkstyle errors.

Signed-off-by: jjlee050 <josephlee050@gmail.com>

gingivitiss committed:
Edit cancelappt, addappt implementation

jjlee050 committed:
Merge pull request #251 from jjlee050/edit-ui

Edit ui

jjlee050 committed:
Update UI class diagram.

Signed-off-by: jjlee050 <josephlee050@gmail.com>

jjlee050 committed:
Change BrowserPanel to BrowserDisplayPanel.

Signed-off-by: jjlee050 <josephlee050@gmail.com>

jjlee050 committed:
Tidy up MainWindow.java

Signed-off-by: jjlee050 <josephlee050@gmail.com>

jjlee050 committed:
Tidy up title screen.

Signed-off-by: jjlee050 <josephlee050@gmail.com>

arsalan0c committed:
fix conflicts

jjlee050 committed:
Tidyup PatientDetailsDisplayPanel and MainWindow.

Signed-off-by: jjlee050 <josephlee050@gmail.com>

jjlee050 committed:
Tidy up CommandBox.java.

Signed-off-by: jjlee050 <josephlee050@gmail.com>

aaronseahyh committed:
Edited string case for MedicineName and MedicineType.

jjlee050 committed:
Tidy up UI classes.

Signed-off-by: jjlee050 <josephlee050@gmail.com>

gingivitiss committed:
fill up appointment section of dg

jjlee050 committed:
Tidy up PasswordPrefixFormatter.java.

Signed-off-by: jjlee050 <josephlee050@gmail.com>

jjlee050 committed:
Set values

Signed-off-by: jjlee050 <josephlee050@gmail.com>

jjlee050 committed:
Fix checkstyle error.

Signed-off-by: jjlee050 <josephlee050@gmail.com>

jjlee050 committed:
Fix checkstyle error.

Signed-off-by: jjlee050 <josephlee050@gmail.com>

aaronseahyh committed:
Minor changes to MedicineCard.

gingivitiss committed:
Merge branch 'master' of https://github.com/gingivitiss/main into ugdg

# Conflicts:
#	src/main/java/seedu/clinicio/ui/BrowserPanel.java
#	src/main/java/seedu/clinicio/ui/MainWindow.java
#	src/main/resources/view/MainWindow.fxml

aaronseahyh committed:
Fix Checkstyle violations.

gingivitiss committed:
Merge branch 'ugdg' of https://github.com/gingivitiss/main into ugdg

# Conflicts:
#	docs/DeveloperGuide.adoc

aaronseahyh committed:
Added PPP.

jjlee050 committed:
Tidy up CommandBox.java

Signed-off-by: jjlee050 <josephlee050@gmail.com>

gingivitiss committed:
Merge remote-tracking branch 'upstream/master' into ugdg

# Conflicts:
#	src/main/java/seedu/clinicio/ui/MainWindow.java
#	src/main/resources/view/MainWindow.fxml

jjlee050 committed:
Edit diagram.

Signed-off-by: jjlee050 <josephlee050@gmail.com>

gingivitiss committed:
Add newline

jjlee050 committed:
LoginCommand.java: Edit message.

Signed-off-by: jjlee050 <josephlee050@gmail.com>

jjlee050 committed:
Merge pull request #268 from jjlee050/tidy-up-ui

Tidy up UI

arsalan0c committed:
update analytics

arsalan0c committed:
Merge branch 'master' of https://github.com/CS2103-AY1819S1-W14-1/main into analytics2

gingivitiss committed:
fix checkstyle

arsalan0c committed:
update consultation

arsalan0c committed:
fix checkstyle

jackedelic committed:
Merge branch 'master' into queue

gingivitiss committed:
Merge branch 'master' into appt-ui

jjlee050 committed:
LoginCommand.java: Edit diagrams.

Signed-off-by: jjlee050 <josephlee050@gmail.com>

arsalan0c committed:
Update analytics (#270)

* update analytics

* update analytics

* update analytics

* update analytics

* update analytics

* update analytics

* update analytics

* clean up

* update analytics

* update analytics

* add files

* update

* chart bug fix

* fix feedback bugs

* update analytics

* update analytics

* fix conflict

* add default constructor

* modularize code

* remove unused import

* refactor plot

* fix errors

* fix errors

* fix errors

* update analytics

* revert test

* update test

* refactor

* fix checkstyle

* update analytics

* update analytics

* update analytics

* refactor analytics

* update analytics

jjlee050 committed:
Edit UI component of Architecture.

Signed-off-by: jjlee050 <josephlee050@gmail.com>

gingivitiss committed:
Fix checkstyle

gingivitiss committed:
Merge pull request #271 from gingivitiss/ugdg

Edit UGDG

jackedelic committed:
merge resolve

gingivitiss committed:
change image file

aaronseahyh committed:
Added MedicineDetailsDisplayPanel.

aaronseahyh committed:
Edits to MedicineCard ui.

arsalan0c committed:
Merge pull request #273 from arsalanc-v2/consultation

update consultation

aaronseahyh committed:
Edits to UI to include MedicineDetailsDisplayPanel.

aaronseahyh committed:
Edit GuiTestAssert.

jackedelic committed:
fix checkstyle

aaronseahyh committed:
Minor edits to MainWindow.

aaronseahyh committed:
Minor edit to AddMedicineCommand.

aaronseahyh committed:
Merge branch 'master' into editMedicineUI

gingivitiss committed:
Merge branch 'master' into appt-ui

aaronseahyh committed:
Set-up RepoSense.

aaronseahyh committed:
Merge pull request #275 from aaronseahyh/editMedicineUI

Edit Medicine UI

aaronseahyh committed:
Merge branch 'master' of https://github.com/CS2103-AY1819S1-W14-1/main

jjlee050 committed:
Edit current implementation.

Signed-off-by: jjlee050 <josephlee050@gmail.com>

jjlee050 committed:
Edit current implementation.

Signed-off-by: jjlee050 <josephlee050@gmail.com>

jjlee050 committed:
Merge pull request #272 from jjlee050/tidy-up-login

Tidy up Logic

jjlee050 committed:
Fix checkstyle errors.

Signed-off-by: jjlee050 <josephlee050@gmail.com>

jjlee050 committed:
Fix checkstyle errors.

Signed-off-by: jjlee050 <josephlee050@gmail.com>

jjlee050 committed:
Edit Ui.png

Signed-off-by: jjlee050 <josephlee050@gmail.com>

jjlee050 committed:
Edit UI walkthrough.

Signed-off-by: jjlee050 <josephlee050@gmail.com>

jjlee050 committed:
Merge pull request #264 from jjlee050/edit-dg

Edit developer guide

jjlee050 committed:
Edit UI walkthrough.

Signed-off-by: jjlee050 <josephlee050@gmail.com>

jjlee050 committed:
Merge pull request #256 from arsalanc-v2/fixbugs2

Update logic class diagram

jjlee050 committed:
Remove logout foor coming in v2.0

Signed-off-by: jjlee050 <josephlee050@gmail.com>

jjlee050 committed:
Edit images for login window before and after.

Signed-off-by: jjlee050 <josephlee050@gmail.com>

jjlee050 committed:
Edit image for logout.

Signed-off-by: jjlee050 <josephlee050@gmail.com>

jjlee050 committed:
Add images for adding patient.

Signed-off-by: jjlee050 <josephlee050@gmail.com>

jjlee050 committed:
Edit grammer errors.

Signed-off-by: jjlee050 <josephlee050@gmail.com>

gingivitiss committed:
refactor appointmentdisplay name and getting staff

gingivitiss committed:
refactor

gingivitiss committed:
Merge branch 'appt-ui' of https://github.com/gingivitiss/main into appt-ui

gingivitiss committed:
Merge branch 'master' into appt-ui

gingivitiss committed:
fix missing bracket

gingivitiss committed:
Merge branch 'appt-ui' of https://github.com/gingivitiss/main into appt-ui

arsalan0c committed:
add analytics tests

gingivitiss committed:
fix checkstyle

jjlee050 committed:
Edit major contributions.

Signed-off-by: jjlee050 <josephlee050@gmail.com>

jjlee050 committed:
Fix checkstyle errors.

Signed-off-by: jjlee050 <josephlee050@gmail.com>

jjlee050 committed:
Edit minor contributions.

Signed-off-by: jjlee050 <josephlee050@gmail.com>

gingivitiss committed:
remove test temporarily

jjlee050 committed:
Edit credits.

Signed-off-by: jjlee050 <josephlee050@gmail.com>

gingivitiss committed:
Merge pull request #274 from gingivitiss/appt-ui

Appt ui

arsalan0c committed:
fix conflicts

jjlee050 committed:
Add sample data.

Signed-off-by: jjlee050 <josephlee050@gmail.com>

jjlee050 committed:
Merge pull request #279 from jjlee050/edit-ppp

Edit PPP

jjlee050 committed:
Fix checkstyle error.

Signed-off-by: jjlee050 <josephlee050@gmail.com>

arsalan0c committed:
fix checkstyle

jjlee050 committed:
Edit current implementation.

Signed-off-by: jjlee050 <josephlee050@gmail.com>

jjlee050 committed:
Fix checkstyle error.

Signed-off-by: jjlee050 <josephlee050@gmail.com>

gingivitiss committed:
edit developer guide

gingivitiss committed:
fix formatting

gingivitiss committed:
Add cli to glossary

jjlee050 committed:
Edit current implementation.

Signed-off-by: jjlee050 <josephlee050@gmail.com>

gingivitiss committed:
change addressbook to clinicio

jjlee050 committed:
Edit current implementation.

Signed-off-by: jjlee050 <josephlee050@gmail.com>

jjlee050 committed:
Merge pull request #280 from arsalanc-v2/analytics2

Add analytics tests

jjlee050 committed:
Merge pull request #281 from jjlee050/add-sampledata

Add sample data.

gingivitiss committed:
Removed instances of addressbook

arsalan0c committed:
remove unnecessary line

arsalan0c committed:
remove unnecessary line

jjlee050 committed:
Edit current implementation.

Signed-off-by: jjlee050 <josephlee050@gmail.com>

jjlee050 committed:
Edit current implementation.

Signed-off-by: jjlee050 <josephlee050@gmail.com>

jjlee050 committed:
Make final changes.

Signed-off-by: jjlee050 <josephlee050@gmail.com>

arsalan0c committed:
fix conflicts

aaronseahyh committed:
Added screenshots for commands.

jjlee050 committed:
Edit position for the roles available.

Signed-off-by: jjlee050 <josephlee050@gmail.com>

aaronseahyh committed:
Updated UserGuide.

jjlee050 committed:
Fix checkstyle error.

Signed-off-by: jjlee050 <josephlee050@gmail.com>

jjlee050 committed:
Add images and edit findpatient.

Signed-off-by: jjlee050 <josephlee050@gmail.com>

jjlee050 committed:
Add listpatient images.

Signed-off-by: jjlee050 <josephlee050@gmail.com>

jackedelic committed:
fix doouble tabs selected

jackedelic committed:
fix typo

jackedelic committed:
resolve conflict

jackedelic committed:
fix style

jjlee050 committed:
Add manual test cases.

Signed-off-by: jjlee050 <josephlee050@gmail.com>

jjlee050 committed:
Merge pull request #278 from jjlee050/edit-ug

Edit User guide.

jjlee050 committed:
Merge pull request #284 from jjlee050/edit-ppp

Make final changes to PPP

jjlee050 committed:
Merge pull request #283 from gingivitiss/ugdg

Ugdg

jjlee050 committed:
Merge pull request #285 from arsalanc-v2/analytics2

Remove unnecessary line

jjlee050 committed:
Fix checkstyle error.

Signed-off-by: jjlee050 <josephlee050@gmail.com>

jjlee050 committed:
Edit ui walkthrough.

Signed-off-by: jjlee050 <josephlee050@gmail.com>

jjlee050 committed:
Merge pull request #282 from jjlee050/edit-dg

Edit developer guide

jjlee050 committed:
Merge pull request #255 from iamjackslayer/queue

Add Switch Tab Event

jjlee050 committed:
Edit developer guide and rmeove class diagram.

Signed-off-by: jjlee050 <josephlee050@gmail.com>

jjlee050 committed:
Merge pull request #286 from jjlee050/edit-ug

Edit ui walkthrough for user guide.

jjlee050 committed:
Merge pull request #287 from jjlee050/edit-dg

Edit developer guide

jjlee050 committed:
Fix broken links and images.

Signed-off-by: jjlee050 <josephlee050@gmail.com>

arsalan0c committed:
add consultation storage

arsalan0c committed:
add consultation storage

arsalan0c committed:
add consultation storage

jjlee050 committed:
Merge pull request #288 from jjlee050/edit-dg

Fix broken links and images.

jackedelic committed:
add enqueue dequeue tests

aaronseahyh committed:
Updated DeveloperGuide.

aaronseahyh committed:
Updated ProjectPortfolio.

aaronseahyh committed:
Edited DG.

jackedelic committed:
fix userguide

jackedelic committed:
Merge branch 'master' into queue

jackedelic committed:
add switch tab event to relevant commands such as listpatient and addpatient

jjlee050 committed:
Merge pull request #291 from iamjackslayer/userguide

fix userguide

jjlee050 committed:
Merge pull request #290 from aaronseahyh/documentation

Documentation

jjlee050 committed:
Merge pull request #289 from arsalanc-v2/analytics2

add consultation storage

aaronseahyh committed:
Merge branch 'master' of https://github.com/CS2103-AY1819S1-W14-1/main

jackedelic committed:
change visible to true for patientlistcard

jackedelic committed:
Merge branch 'listpatient' into queue

jjlee050 committed:
Merge pull request #292 from iamjackslayer/listpatient

Listpatient

gingivitiss committed:
Edit ppp

jjlee050 committed:
Merge pull request #293 from iamjackslayer/queue

Improve test coverage

jjlee050 committed:
Merge pull request #294 from gingivitiss/ugdg

Edit ppp

aaronseahyh committed:
Added RepoSense link.

aaronseahyh committed:
Merge pull request #295 from aaronseahyh/editPPP

Edit ppp

arsalan0c committed:
edit author tag

arsalan0c committed:
Merge pull request #296 from arsalanc-v2/analytics4

Reposense edit

jjlee050 committed:
Remove all @@author tags.

Remove all @@author tags in ClinicIO.java, Model.java, ModelManager.java, XmlAdaptedPatient.java, ClinicIoTest.java, ModelManagerTest.java.

Signed-off-by: jjlee050 <josephlee050@gmail.com>

jjlee050 committed:
Revert "Remove all @@author tags."

This reverts commit 5a760ea

Signed-off-by: jjlee050 <josephlee050@gmail.com>

jjlee050 committed:
Edit author tags.

Remove author tags in MainWindow.java and XmlAdaptedPatient.java.

Signed-off-by: jjlee050 <josephlee050@gmail.com>

jjlee050 committed:
Add author tags to fix reposense codeview.

Signed-off-by: jjlee050 <josephlee050@gmail.com>

jjlee050 committed:
Add necessary @@author tags to fix Reposense.

Signed-off-by: jjlee050 <josephlee050@gmail.com>

jjlee050 committed:
Remove author tags for methods that I did not make.

Signed-off-by: jjlee050 <josephlee050@gmail.com>

jjlee050 committed:
Edit tags in Model.java

Signed-off-by: jjlee050 <josephlee050@gmail.com>

aaronseahyh committed:
Merge RepoSense PR.

aaronseahyh committed:
Add author tags.

jjlee050 committed:
Merge pull request #297 from jjlee050/edit-reposense-tags

Fix Reposense author tags
